modules/ROOT/pages/upgrade-cluster.adoc

```diff
@@ -163,20 +163,20 @@ Perform the following steps to resolve common errors:
 . Use the `gravity plan` command to identify the phase in which the upgrade paused: 
 +
 ----
-sudo ./gravity plan
+sudo gravity plan
```

Comment:
same comment as above (which gravity binary to use in the case of upgrades)

Reply:
in this case the new version from the `installer` directory as that is the one running the upgrade